.changeset/smooth-rings-mate.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/theme': patch
+---
+
+Added support for string `fontSize` values (e.g. `"2.5rem"`) in themes in addition to numbers. Also added an optional `fontFamily` prop for header typography variants to allow further customization.
```

Comment:
I'm wondering if it would make sense to also add some documentation about how to use this overall? That might go above this section in the docs: https://github.com/backstage/backstage/blob/master/docs/getting-started/app-custom-theme.md#overriding-backstage-and-material-ui-components-styles